### PR TITLE
Restructure | Bump RET & Scrypto & UniFFI version | Add CI (test.yml)

### PR DESCRIPTION
**Pull request by CyonAlexRDX**
* Bump RET version
* Bump Scrypto version (same as used by new RET)
* Remove unneeded crate dependency `transaction` (part of Scrypto)
* Remove Workspace => mono crate (already was mono crate)
* Restructure files
* Changed naming convention of imported Radix types to `Scrypto` prefix for types from radixdlt/radixdlt-scrypto package and `RET` prefix for types from radix-engine-toolkit package.
* Replace `rust.yml` with `test.yml` that is based on the same [contents as RET](https://github.com/radixdlt/radix-engine-toolkit/blob/main/.github/workflows/test.yml), but with `tarpaulin` and `upload to codecov` added

### CODE REVIEW

File: uniffi.toml
Comment (CyonAlexRDX) [inline uniffi.toml:1]: unchanged, just moved and removed Python bindings...

**Review by GhenadieVP — APPROVED** [review]

File: src/profile/v100/networks/network/network_id.rs [hunk 1]
```diff
@@ -114,40 +114,40 @@ impl std::fmt::Display for NetworkID {
...
-            Zabanet => NativeNetworkDefinition::zabanet(),
-            Enkinet => NativeNetworkDefinition {
+            Mainnet => ScryptoNetworkDefinition::mainnet(),
+            Stokenet => ScryptoNetworkDefinition::stokenet(),
+            Adapanet => ScryptoNetworkDefinition::adapanet(),
+            Nebunet => ScryptoNetworkDefinition::nebunet(),
+            Kisharnet => ScryptoNetworkDefinition::kisharnet(),
+            Ansharnet => ScryptoNetworkDefinition::ansharnet(),
+            Zabanet => ScryptoNetworkDefinition::zabanet(),
+            Enkinet => ScryptoNetworkDefinition {
```
Comment (micbakos-rdx) [inline src/profile/v100/networks/network/network_id.rs:130]: Don't such definitions exist in RET? This is why you added some custom ones?

File: uniffi.toml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+[bindings.swift]
+module_name = "Sargon"
+
+[bindings.kotlin]
+package_name = "sargon"
```
Comment (micbakos-rdx) [inline uniffi.toml:5]: 👍

File: src/profile/v100/networks/network/network_id.rs (hunk 1 above)
Comment (Sajjon) [reply src/profile/v100/networks/network/network_id.rs:130]: mm does not exist

**Pull request merged**